### PR TITLE
Added names for fields based on strings in class_5315

### PR DESCRIPTION
**Pull request by Kroppeb**
Seems to store level metadata?

### CODE REVIEW

File: mappings/net/minecraft/class_5315.mapping [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+		ARG 1 version
+		ARG 2 lastPlayed
+		ARG 4 versionName
+		ARG 5 worldVersion
+		ARG 6 stable
+	METHOD method_29022 getVersion ()I
+	METHOD method_29023 fromDynamic (Lcom/mojang/serialization/Dynamic;)Lnet/minecraft/class_5315;
+	METHOD method_29024 getLastPlayed ()J
+	METHOD method_29025 getVersionName ()Ljava/lang/String;
+	METHOD method_29026 getWorldVersion ()I
```
Comment (liach) [inline mappings/net/minecraft/class_5315.mapping]: should we prefer "data version" or "world version" name?

File: mappings/net/minecraft/class_5315.mapping [hunk 2]
```diff
@@ -0,0 +1,18 @@
+CLASS net/minecraft/class_5315
+	FIELD field_25024 version I
```
Comment (liach) [inline mappings/net/minecraft/class_5315.mapping]: Might specify this is the version of level.dat

File: mappings/net/minecraft/class_5315.mapping [hunk 3]
```diff
@@ -0,0 +1,18 @@
+CLASS net/minecraft/class_5315
```
Comment (liach) [inline mappings/net/minecraft/class_5315.mapping]: This is like `SaveVersionInfo` imo

File: mappings/net/minecraft/class_5315.mapping (hunk 2 above)
Comment (Kroppeb) [reply mappings/net/minecraft/class_5315.mapping]: What do you mean exactly?

File: mappings/net/minecraft/class_5315.mapping (hunk 1 above)
Comment (i509VCB) [reply mappings/net/minecraft/class_5315.mapping]: Hmm, what is it called in `MinecraftVersion`?

File: mappings/net/minecraft/class_5315.mapping (hunk 2 above)
Comment (liach) [reply mappings/net/minecraft/class_5315.mapping]: this is the level.dat format version, not the data version of the save represented by the level.dat

File: mappings/net/minecraft/class_5315.mapping (hunk 1 above)
Comment (liach) [reply mappings/net/minecraft/class_5315.mapping]: `world_version` in `version.json`, but it is called `DataVersion` in like persistent data, level.dat, etc, and the community uses "data version" as a convention already. Imo the `world_version` is not accurate, as it should rather be "save version" if we follow this logic.

**Comment by liach** [comment]
Mind edit this pr to merge into 1.16-pre2

**Review by liach — APPROVED** [review]

**Review by modmuss50 — APPROVED** [review]

**Review by YanisBft — APPROVED** [review]

**Pull request merged**